libff/algebra/curves/alt_bn128/alt_bn128_g1.cpp

```diff
@@ -9,6 +9,8 @@
 
 namespace libff {
 
+using std::size_t;
```

Comment:
Should we be doing this more often in the codebase?

Reply:
At some point I noticed that whether we use `std::size_t` is inconsistent, and I think professor Chiesa agreed we can only use `std::size_t`. I saw that for existing files .hpp files used `std::` everywhere and .cpp files use a `using std::size_t;` statement, so I followed that convention for any files I edit. I just got round to doing that for all files in curves/